### ISSUE TITLE
How to disable components in local-ai-packaged (because already installed locally)

### ISSUE THREAD

**Issue by nofrillsdev** [coleam00/ai-agents-masterclass#13]
I already have a setup of Ollama and OpenWebUI in my Ubuntu WSL instance. Because of limitations of my corporate notebook enabling Cuda support in Docker is not possible. 

So I need to have Ollama be run in WSL to use my eGPU.

Is there a way to custimze the setup in the folder https://github.com/coleam00/ai-agents-masterclass/tree/main/local-ai-packaged so I can reuse the existing Ollama and OpenWebUI installations?

**Comment by jordanurbs** [comment]
Did you solve this?

I can't run the initial docker compose becaues I already have open-webui installed.

**Comment by ghost** [comment]
Yes, it´s actually quite easy - just delete the corresponding section (in my case the Ollama section). You only have to make sure that the ports specified in the Docker compose file match your local installation of OpenWebUI.
